src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { useTabsContext } from "./tabs-context";
+
+interface TabPanelProps {
+	id: string;
```

Comment:
rename `id` to `value` which is consistent with `<select/>`'s `<option>` element the closest HTML counterpart to Tabs.